proposals/0023-consensus-and-transaction-inclusion-verification.md

```diff
@@ -12,48 +12,26 @@ created:  2023-05-25
 
-This is where diet clients come in, users run the client to verify confirmation of their transaction without trusting the RPC. The SIMD is the first step towards implementing the diet client by proposing a small change to the rpc service that allows the client to validate if supermajority stake actually signed off on a block. 
+Currently, for a user to validate whether their transaction was included in a block, it needs to trust the confirmation from an RPC provider. This is a glaring attack vector for malicious RPC actors that could lie to users if it's in their own interest. 
 
-This ensures that at-least the user doesn't have to trust the RPC service that is centralised and can rather trust the supermajority of the network which is less probable to be corrupt than a malicious RPC. However it is not impossible, hence the full diet client implementation discusses further steps to counter that and this is only the consensus verifying stage of the client.
+To combat this, a user must run a full node to process the entire ledger and verify the blocks and votes themselves. The downside of this is that its very costly to run a full node which makes inaccessible to everyday users. 
 
+One solution to this problem is to use diet clients to verify transaction confirmations without fully trusting the RPC. This SIMD is the first step towards implementing diet clients for Solana and provides a way for users to verify their transaction is included in a block and that the block has recieved a supermajority of votes.
 
 ## Alternatives Considered
-Another solution is to use the gossip plane to read votes out of the CRDS optimistically and then confirm by verifying inclusion with the blockhash. The advantage being that no changes are required to the validator to read the votes. However this has a couple of drawbacks:
-- Votes have a high probability of getting dropped after being inserted into the CRDS as sometimes validators can vote on older slots after voting on the newer slot.
-- Validators aren't obligated to propagate gossip changes and have reason to not do so as it reduces egress costs.
-
-We could also just directly make these calls from the client itself but its much faster and more convenient to do it on server side.
 
+None
 
 ## New Terminology
 
-TransactionProof: A structure containing necessary information to verify if a transaction was included in the bank hash of a slot.
-
-## Detailed Design
-The protocol interaction will be as follows:
```

Comment:
this isnt necessary imo - the full steps of using the new RPC method are already described below using psuedocode 

discussing reading validator sets, stake weights history, parsing vote signatures is not related to implementing the new RPC method (its probs a separate SIMD imo) so we dont want to spend too much time on it (just a sentence or two)

Reply:
same as explained before